documentation/src/main/antora/modules/ROOT/pages/tutorial-create-my-first-component.adoc

```diff
@@ -0,0 +1,165 @@
+
+1. Save the job created earlier and close Talend Studio.
+2. Back in IntelliJ open the *LoggerOutput* class. This is the class where the component logic can be defined.
+3. Look for the `@ElementListener` method. It is already present and references the default input branch that was defined in the starter, but it is not complete yet.
+4. To be able to log the data in input to the console, add the following lines: +
++
+[source,java,indent=0,subs="verbatim,quotes,attributes"]
+----
+//Log to the console
+        System.out.println("["+configuration.getLevel().toUpperCase()+"] "+defaultInput);
+----
++
+The `@ElementListener` method now looks as follows: +
++
+[source,java,indent=0,subs="verbatim,quotes,attributes"]
+----
+@ElementListener
+    public void onNext(
+            @Input final JsonObject defaultInput) {
+        // this is the method allowing you to handle the input(s) and emit the output(s)
+        // after some custom logic you put here, to send a value to next element you can use an
+        // output parameter and call emit(value).
+
+        //Log to the console
+        System.out.println("["+configuration.getLevel().toUpperCase()+"] "+defaultInput);
+    }
+----
+[start="5"]
+. Open the Terminal again to compile the project and deploy the component again. To do that, run successively the two following commands:
+** `mvnw clean install`
+** ``mvnw talend-component:deploy-in-studio -Dtalend.component.studioHome="<path to Talend Studio home>"`
+
+The update of the component logic should now be deployed to the Studio. You are now ready to build a job and use your component for the first time.
+
+To learn the different possibilities and methods available to develop more complex logics, refer to xref:component-define-processor-output.adoc[this document].
+
+If you want to avoid having to close and re-open Talend Studio every time you need to make an edit, you can enable the developer mode, as explained in xref:studio.adoc#developer-mode[this document].
+
+== Building a job with the component
+As the component is now ready to be used, it is time to create a job and check that it behaves as intended.
+
+1. Open Talend Studio again and go to the job created earlier. The new component is still there.
+2. Add a *tRowGenerator* component and connect it to the logger.
+3. Double-click the *tRowGenerator* to specify the data to generate:
+** Add a first column named `firstName` and select the *TalendDataGenerator.getFirstName() function.
+** Add a second column named 'lastName' and select the *TalendDataGenerator.getLastName() function.
+** Set the *Number of Rows for RowGenerator* to `10`. +
++
+image::tutorial_build_job_trowgenerator.png[tRowGenerator]
+4. Validate the *tRowGenerator* configuration.
+5. Open the *TutorialFamilyLogger* component and set the *level* field to `my_data`. +
```

Comment:
can you please replace **my_data** to INFO, DEBUG or TRACE as they are known concepts of Loggers